### PR TITLE
External auth review fixes

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: topics/admin/tutorials/external-auth/tutorial.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ For this exercise we will use a basic password file method for authenticating -
 > 1. Edit your `galaxyservers` group variables file and update the main location block defined for serving galaxy. Add the parameters:
```
Comment (mvdbeek) [inline topics/admin/tutorials/external-auth/tutorial.md]: @hexylena should this happen in the galaxy.j2 template ?

Comment (hexylena) [reply topics/admin/tutorials/external-auth/tutorial.md]: yes, it should. a previous version used a different nginx role where the config was embedded

Comment (mvdbeek) [reply topics/admin/tutorials/external-auth/tutorial.md]: Cool, thanks for confirminig

**Comment by mvdbeek** [comment]
This should be ready for review and worked fine with these changes.

File: topics/admin/tutorials/external-auth/tutorial.md [hunk 2]
```diff
@@ -98,7 +99,7 @@ For this exercise we will use a basic password file method for authenticating -
 >      galaxy:
 >        ...
 >        use_remote_user: true
->        remote_user_maildomain: "{{ hostname }}"
+>        remote_user_maildomain: "{% raw %}{{ hostname }}{% endraw %}"
```
Comment (hexylena) [inline topics/admin/tutorials/external-auth/tutorial.md]: does this produce a full enough hostname to work? With the .eu? Or just `gcc-8` or so?

Comment (hexylena) [reply topics/admin/tutorials/external-auth/tutorial.md]:
```suggestion
>        remote_user_maildomain: "{% raw %}{{ inventory_hostname }}{% endraw %}"
```

**Pull request merged**